package.json

```diff
@@ -27,8 +27,10 @@
   },
   "dependencies": {
     "@expo/vector-icons": "^12.0.0",
+    "@react-native-community/async-storage": "^1.12.1",
```

Comment:
I think there might be an issue with this package along with expo.
https://stackoverflow.com/questions/56029007/nativemodule-asyncstorage-is-null-with-rnc-asyncstorage

Getting an error where Async Storage is null.
See attached:

![image](https://user-images.githubusercontent.com/35586281/144366330-d47777ae-cf96-43df-ae9e-3fc51479ec27.png)

Reply:
Good catch -- updated to https://react-native-async-storage.github.io/async-storage/docs/usage/